core/shaper/shaper.go

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package shaper
+
+import (
+	"net"
+	"os/exec"
+	"strconv"
+
+	"github.com/magefile/mage/sh"
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/mysteriumnetwork/node/eventbus"
+	"github.com/mysteriumnetwork/node/logconfig"
+	"github.com/mysteriumnetwork/node/services/shared"
+	"github.com/pkg/errors"
+)
+
+var log = logconfig.NewLogger()
+
+const (
+	limitKbps = 5000
```

Comment:
I remember, we talked about default of 6Mbps

Reply:
After reconsidering it, 5Mbps should be enough for Netflix.